submissions/cynthiaNwakaeme/php-database/cookies.php

```diff
@@ -1,55 +1,69 @@
+            
+            /* GENERATING COOKIES
+            1. result is initalized to get all records from database
+            2. This result is serialize to change the array/object to simple string/text file which is required to create cookies.
+             */
+             $result = $stmt->fetchAll();
+            setcookie("customer", serialize($result), time() + (86400 * 30), "/", "", false); //60sec *60mins *24hours/day => 86400 = 1 day   
+            return $result;
+            
+        }catch (PDOException $err) {
+                die('ERROR: Could not execute' .$sql . $err->getMessage());
+        }
+    }
+}
 
-    /* GENERATING COOKIES
-    1. result is initalized to get all records from database
-    2. This result is serialize to change the array/object to simple string/text file which is required to create cookies.
-     */
-     $result = $stmt->fetchAll();
-    setcookie("customer", serialize($result), time() + (86400 * 30), "/", "", false); //60sec *60mins *24hours/day => 86400 = 1 day
+//  Creating a new instance of the class customer so as to have access to all its properties
+$customer = new Customer();
 
-    // To display results on the browser as a table, we convert the result to array/object.
+ // To display results on the browser as a table, we convert the result to array/object.
+//  $output;
+ if(isset($_COOKIE['customer'])){
     $output = unserialize($_COOKIE["customer"]);
+ }else{
+    $output = $customer->dbQuery();
+ }
+    
 
-    if(count($output) > 0){
-        echo"<table>";
-            echo"<tr>";
-                echo"<th> S/N <th>";
-                echo"<th> Full Name <th>";
-                echo"<th> Email Address <th>";
-                echo"<th> Created At <th>";
-                echo"<th> Actions <th>";
-            echo"</tr>";
-        // Loop through to access records (Objects in an array i.e datatype <=> JSON formatt)
-        foreach($output as $row){
-            echo"<tr>";
-                echo "<td>" . $row['id'] ."</td>";
-                echo "<td>" . $row['full_name'] ."</td>";
-                echo "<td>" . $row['email_address'] ."</td>";
-                echo "<td>" . $row['created_at'] ."</td>";
-                echo "<td>" . '<button>View</button>' . "</td>";
-            echo"</tr>";
-        }
-        echo"<table>";
-
-        // Close/Free result set
-        unset($output);
-    }else{
-        echo 'No records matching your query were found.';
+if(count($output) > 0){
+    echo"<table>";
+        echo"<tr>";
+            echo"<th> S/N </th>";
+            echo"<th> Full Name </th>";
+            echo"<th> Email Address </th>";
+            echo"<th> Created At </th>";
+            echo"<th> Actions </th>";
+        echo"</tr>";
+    // Loop through to access records (Objects in an array i.e datatype <=> JSON formatt)
+    foreach($output as $row){
+        echo"<tr>";
+            echo "<td>" . $row['id'] ."</td>";
+            echo "<td>" . $row['full_name'] ."</td>";

```

Comment:
Yout customers table does not have any column called `full_name` 
![image](https://user-images.githubusercontent.com/43061141/106471690-3022d200-64a2-11eb-850f-1e910ec6b817.png)

Get the result for Full name from `first_name` and `last_name`

Reply:
Hello @adaugochi , I am so sorry i am reverting late. I only just figured out that I had to use the existing database. I actually created another for this task.

Thank you for pointing me to that. Corrections has been effected. Kindly review.

Reply:
Also, I noticed there is no created_at in the customers table so I used the one in the orders table